src/main/java/automaticlotto/domain/Lottos.java

```diff
@@ -0,0 +1,37 @@
+package automaticlotto.domain;
+
+import automaticlotto.automaticexception.ExceptionCommand;
+import automaticlotto.automaticexception.InputValueException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Lottos {
+
+    private List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        validateLottos(lottos);
+        this.lottos = lottos;
+    }
+
+    private void validateLottos(List<Lotto> lottos) {
+        if (lottos == null || lottos.isEmpty()) {
+            throw new InputValueException(ExceptionCommand.LOTTOS_EMPTY_EXCEPTION_MESSAGE);
+        }
+    }
+
+    public static Lottos createLottos(int buyLottoNumbers) {
+        List<Lotto> lottos = new ArrayList<>();
+        Lotto lotto;
+        for (int i = 0; i < buyLottoNumbers; i++) {
+            lotto = new Lotto(Lotto.sortLotto(Lotto.createLotto()));
+            lottos.add(lotto);
```

Comment:
```suggestion
            lottos.add(new Lotto(Lotto.sortLotto(Lotto.createLotto())););
```
변수를 사용할 일이 없으면, 위와 같이 바로 lottos에 추가해도 좋을 것 같네요 :)